src/Microsoft.DotNet.CodeFormatting.Tests/AnalyzerFixerTestBase.cs

```diff
@@ -12,10 +12,31 @@ public abstract class AnalyzerFixerTestBase : CodeFormattingTestBase
     { 
         private IFormattingEngine _engine; 
 
-        protected AnalyzerFixerTestBase(IFormattingEngine engine)
```

Comment:
Previously, I had each derived class create an engine and pass it down to the base class through this ctor. But the derived classes don't need the engine, so let the base class take care of it. This simplifies the derived classes.